algorithms.py

```diff
@@ -307,7 +333,8 @@ def __init__(
             population_growth_rate_changes, population_size_changes,
             migration_matrix_element_changes, bottlenecks, model='hudson',
             from_ts=None, max_segments=100, num_labels=1, sweep_trajectory=None,
-            full_arg=False, time_slice=None):
+            full_arg=False, time_slice=None, geneconversion_rate=0.0,
```

Comment:
Let's change this to ``gene_conversion_rate``  and ``gene_conversion_length`` for consistency please.

Reply:
done